assets/js/components/indicators.js

```diff
@@ -263,12 +263,18 @@ export class IndicatorSection {
     $provinceButton.on('click', (() => {
       this.chart.loadMedians(this.formatMedians().provincial);
       ga('send', 'event', 'chart-averages', `${this.key} provincial`);
+      gtag('event', 'chart_averages', {
+        chart: `${this.key} provincial`,
+      });
     }));
 
     const $nationalButton = $(' <button class="button" style="display: unset">nationally</button>');
     $nationalButton.on('click', (() => {
       this.chart.loadMedians(this.formatMedians().national);
       ga('send', 'event', 'chart-averages', `${this.key} national`);
+      gtag('event', 'chart_averages', {
```

Comment:
@michaelglenister is there a reason the `ga` is `national` but the `gtag` is `provincial`?

Reply:
Copy pasta error, fixed